Userland/Libraries/LibCore/FileStream.h

```diff
@@ -44,6 +44,13 @@ class InputFileStream final : public InputStream {
         return buffer.bytes().copy_to(bytes);
     }
 
```

Comment:
The commit category for this should be `LibCore`, and "add new member function" isn't particularly informative
how about `LibCore: Add fallible FileStream::try_read() member function`?